<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>809</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login </string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(31, 155, 93);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(35, 178, 106, 255), stop:1 rgba(120, 191, 155, 255));
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>461</width>
        <height>551</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>461</width>
        <height>551</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget {
    border-radius: 12px;        /* Radius to make the borders rounded */
    background-color: rgb(0, 0, 0);
}
</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>-10</x>
         <y>30</y>
         <width>497</width>
         <height>423</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="rightMargin">
           <number>5</number>
          </property>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>98</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="Welcome_Label">
            <property name="minimumSize">
             <size>
              <width>271</width>
              <height>171</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>271</width>
              <height>171</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel
{
color : #83ffbd;
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="resources.qrc">:/Icons/Icons/WhiteIcon.png</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>108</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="leftMargin">
           <number>60</number>
          </property>
          <property name="topMargin">
           <number>30</number>
          </property>
          <item>
           <widget class="QLabel" name="Error_Label">
            <property name="minimumSize">
             <size>
              <width>381</width>
              <height>31</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>381</width>
              <height>31</height>
             </size>
            </property>
            <property name="font">
             <font>
              <italic>false</italic>
              <bold>false</bold>
              <underline>false</underline>
              <strikeout>false</strikeout>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color : white;
background-color: rgb(255, 0, 0);
padding : 5px;
</string>
            </property>
            <property name="text">
             <string>Incorrect Username or Password.</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="leftMargin">
             <number>40</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <property name="bottomMargin">
               <number>10</number>
              </property>
              <item>
               <widget class="QLineEdit" name="Username_LineEdit">
                <property name="minimumSize">
                 <size>
                  <width>291</width>
                  <height>41</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>291</width>
                  <height>41</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(255, 255, 255);
padding : 7px;
color : black;</string>
                </property>
                <property name="placeholderText">
                 <string>Username</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="Password_LineEdit">
                <property name="minimumSize">
                 <size>
                  <width>291</width>
                  <height>41</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>291</width>
                  <height>41</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(255, 255, 255);
padding : 7px;
color : black;</string>
                </property>
                <property name="echoMode">
                 <enum>QLineEdit::EchoMode::Password</enum>
                </property>
                <property name="placeholderText">
                 <string>Password</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="SignIn_PushButton">
              <property name="minimumSize">
               <size>
                <width>291</width>
                <height>41</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>291</width>
                <height>41</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{
background-color: #83ffbd;
	color: rgb(0, 0, 0);
}

QPushButton:hover{
	background-color: rgb(46, 194, 126);
}

QPushButton:pressed{
	background-color: rgb(33, 116, 77);
}
</string>
              </property>
              <property name="text">
               <string>LOG IN</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="Error_Label_2">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>550</y>
         <width>381</width>
         <height>31</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>381</width>
         <height>31</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>381</width>
         <height>31</height>
        </size>
       </property>
       <property name="font">
        <font>
         <italic>false</italic>
         <bold>false</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color : white;
background-color: rgb(255, 0, 0);
padding : 5px;
</string>
       </property>
       <property name="text">
        <string>Incorrect Username or Password.</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>809</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
